state/validatorsInfoMap.go

```diff
@@ -0,0 +1,224 @@
+package state
+
+import (
+	"bytes"
+	"encoding/hex"
+	"fmt"
+	"sync"
+
+	"github.com/ElrondNetwork/elrond-go-core/core/check"
+)
+
+type shardValidatorsInfoMap struct {
+	mutex      sync.RWMutex
+	valInfoMap map[uint32][]ValidatorInfoHandler
+}
+
+// NewShardValidatorsInfoMap creates an instance of shardValidatorsInfoMap which manages a
+// <shardID, validatorsInfo> map internally
+func NewShardValidatorsInfoMap(numOfShards uint32) *shardValidatorsInfoMap {
```

Comment:
you can simply this as not providing the numOfShards
we do not need a size for a map variable

Reply:
I was thinking the same, but was not sure. I'll delete it, thanks for suggestion !